bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -189,37 +140,22 @@ <h3>Hubs</h3>
     <li>hubs.mozilla.com</li>
     <li>reticulum.io</li>
   </ul>
-  
+
   <h3>Localization</h3>
   <ul class="mzp-u-list-styled">
-    <li>l10n.mozilla.org</li>
     <li>pontoon.mozilla.org</li>
   </ul>
 
   <h3>Observatory</h3>
   <ul class="mzp-u-list-styled">
-    <li>http-observatory.security.mozilla.org</li>
     <li>observatory.mozilla.org</li>
     <li>tls-observatory.services.mozilla.com</li>
   </ul>
 
-  <h3>Payment Subscription</h3>
-  <ul class="mzp-u-list-styled">
-    <li>prod.fxa.mozilla-subhub.app</li>
-    <li>subscriptions.firefox.com</li>
-  </ul>
-
   <h3>Pocket</h3>
   <ul class="mzp-u-list-styled">
     <li>getpocket.com</li>
     <li>api.getpocket.com</li>
-    <li>app.getpocket.com</li>
-    <li>widgets.getpocket.com</li>
-  </ul>
-
-  <h3>Premium Services</h3>
-  <ul class="mzp-u-list-styled">
-    <li>premium.firefox.com</li>
```

Comment:
This was maybe an older site that was replaced by subscriptions.firefox.com ? I found the site's repo but it hasn't been updated in a long time. If you load the domain you get redirected to the /vpn/ path on www.mozilla.org, but if you load the IP address you get redirected to https://fpn.firefox.com

Reply:
so we can possibly remove this right? the fpn.firefox.com is already included

Reply:
yeah

Reply:
by the way, it was mentioned that FPN will also be deprecated this year